<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Settings</class>
    <widget class="QWidget" name="Settings">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>592</width>
                <height>544</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QGroupBox" name="ClientSettingsBox">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>260</y>
                    <width>161</width>
                    <height>211</height>
                </rect>
            </property>
            <property name="title">
                <string>Client Settings</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <widget class="QPushButton" name="WizardButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>30</y>
                        <width>141</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Add Games to Horizon</string>
                </property>
            </widget>
            <widget class="QPushButton" name="ClearDatabaseButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>60</y>
                        <width>141</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Clear Database</string>
                </property>
            </widget>
            <widget class="QPushButton" name="ClearLaunch">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>90</y>
                        <width>141</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Clear Launch Options</string>
                </property>
            </widget>
        </widget>
        <widget class="QGroupBox" name="StyleSettingsBox">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>260</y>
                    <width>700</width>
                    <height>211</height>
                </rect>
            </property>
            <property name="title">
                <string>Style Settings</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>30</y>
                        <width>141</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="autoFillBackground">
                    <bool>false</bool>
                </property>
                <property name="text">
                    <string>Colour Options:</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>50</y>
                        <width>110</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Body Background</string>
                </property>
            </widget>
            <widget class="QPushButton" name="BodyColor">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>50</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QPushButton" name="NavbarBG">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>80</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_4">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>80</y>
                        <width>125</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Navbar Background</string>
                </property>
            </widget>
            <widget class="QPushButton" name="NavbarHover">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>110</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_5">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>110</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Navbar Hover</string>
                </property>
            </widget>
            <widget class="QPushButton" name="NavbarSelected">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>140</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_6">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>140</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Navbar Selected</string>
                </property>
            </widget>
            <!-- COLUMN 2 -->
            <widget class="QPushButton" name="TitleBarColor">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>50</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_7">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>50</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Titlebar Color</string>
                </property>
            </widget>
            <widget class="QPushButton" name="ActiveElement">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>80</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_8">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>80</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Active Element</string>
                </property>
            </widget>
            <widget class="QPushButton" name="InactiveSelection">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>110</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_9">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>110</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Inactive Selection</string>
                </property>
            </widget>
            <widget class="QPushButton" name="HoverSelection">
                <property name="geometry">
                    <rect>
                        <x>180</x>
                        <y>140</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_10">
                <property name="geometry">
                    <rect>
                        <x>210</x>
                        <y>140</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Hover Selection</string>
                </property>
            </widget>
            <!-- COLUMN 3 -->
            <widget class="QPushButton" name="DarkElement">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>50</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_11">
                <property name="geometry">
                    <rect>
                        <x>380</x>
                        <y>50</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Dark Element</string>
                </property>
            </widget>
            <widget class="QPushButton" name="LightText">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>80</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_12">
                <property name="geometry">
                    <rect>
                        <x>380</x>
                        <y>80</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Light Text</string>
                </property>
            </widget>
            <widget class="QPushButton" name="DarkText">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>110</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_13">
                <property name="geometry">
                    <rect>
                        <x>380</x>
                        <y>110</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Dark Text</string>
                </property>
            </widget>
            <widget class="QPushButton" name="SubText">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>140</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_14">
                <property name="geometry">
                    <rect>
                        <x>380</x>
                        <y>140</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Sub Text</string>
                </property>
            </widget>
            <!-- COLUMN 4 -->
            <widget class="QPushButton" name="PrimaryBase">
                <property name="geometry">
                    <rect>
                        <x>525</x>
                        <y>50</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_15">
                <property name="geometry">
                    <rect>
                        <x>555</x>
                        <y>50</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Primary Base</string>
                </property>
            </widget>
            <widget class="QPushButton" name="SecondaryBase">
                <property name="geometry">
                    <rect>
                        <x>525</x>
                        <y>80</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_16">
                <property name="geometry">
                    <rect>
                        <x>555</x>
                        <y>80</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Secondary Base</string>
                </property>
            </widget>
            <widget class="QPushButton" name="TertiaryBase">
                <property name="geometry">
                    <rect>
                        <x>525</x>
                        <y>110</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_17">
                <property name="geometry">
                    <rect>
                        <x>555</x>
                        <y>110</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Tertiary Base</string>
                </property>
            </widget>
            <widget class="QPushButton" name="DarkestBase">
                <property name="geometry">
                    <rect>
                        <x>525</x>
                        <y>140</y>
                        <width>21</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label_18">
                <property name="geometry">
                    <rect>
                        <x>555</x>
                        <y>140</y>
                        <width>110</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Darkest Base</string>
                </property>
            </widget>
            <widget class="QPushButton" name="ResetColors">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>180</y>
                        <width>141</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Reset Colours to Default</string>
                </property>
            </widget>
        </widget>
        <widget class="QGroupBox" name="UserSettingsBox">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>541</width>
                    <height>231</height>
                </rect>
            </property>
            <property name="title">
                <string>User Settings</string>
            </property>
            <widget class="QPushButton" name="ChangeUsernameButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>30</y>
                        <width>111</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Change Username</string>
                </property>
            </widget>
            <widget class="QPushButton" name="ChangeUsernameButton_2">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>60</y>
                        <width>111</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Change Password</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
